### PR TITLE
included extern definition of global variable EPC_ST_ERR_COUNTER

### PR DESCRIPTION
**Pull request by pastorpflores**

### CODE REVIEW

File: firmware/STM32/Core/Src/main.c [hunk 1]
```diff
@@ -17,6 +17,7 @@
   */
 	#include "epc_conf.h"
 	#include "epc_st_err.h" //Import EPC_ST_ERR_COUNTER
+	extern uint8_t EPC_ST_ERR_COUNTER;
```
Comment (mariuscrsn) [inline firmware/STM32/Core/Src/main.c:20]: Move these 3 lines to else block: below of include #include "main.h"

File: firmware/Sources/HAL/HAL_STS/hal_sts.c
Comment (mariuscrsn) [inline firmware/Sources/HAL/HAL_STS/hal_sts.c:1]: Move #include "hal_sts.h" to       Include headers of the component

File: firmware/Sources/HAL/HAL_SYS/hal_sys.c
Comment (mariuscrsn) [inline firmware/Sources/HAL/HAL_SYS/hal_sys.c:1]: Change #include "stm32f3xx_hal.h" to main because we use systemclock from main

**Review by mariuscrsn — CHANGES_REQUESTED** [review]

File: firmware/STM32/Core/Src/main.c (hunk 1 above)
Comment (pastorpflores) [reply firmware/STM32/Core/Src/main.c:20]: moved except `#include "epc_conf.h"`

**Review by mariuscrsn — APPROVED** [review]

**Pull request merged**